### PR TITLE
[Site Modules] Manifest file filters, part 3

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes
This adds filter attribute to some of the fields in manifest files.

### Testing Instructions
Code review.

### Documentation Changes Required
No.

### CODE REVIEW

File: modules/mod_wrapper/helper.php [hunk 1]
```diff
@@ -30,9 +30,9 @@ public static function getParams(&$params)
 		$params->def('url', '');
 		$params->def('scrolling', 'auto');
 		$params->def('height', '200');
```
Comment (Quy) [inline modules/mod_wrapper/helper.php]: `height` is a number with a default value of `500`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 63fb6f24c32258594cb4782b6df8ca607205f187<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20857">issues.joomla.org/tracker/joomla-cms/20857</a>.</sub>

**Comment by carlitorweb** [comment]
I have tested this item :white_check_mark: successfully on 63fb6f24c32258594cb4782b6df8ca607205f187<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20857">issues.joomla.org/tracker/joomla-cms/20857</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20857">issues.joomla.org/tracker/joomla-cms/20857</a>.</sub>

**Pull request merged**